### DISCUSSION TITLE
Why it doesn't need to substract 1 in flip operation?

### DISCUSSION THREAD

**Discussion by rockywind** [open-mmlab/mmdetection#9166] [Q&A] [answered]
![6d812ad505c2456752c13688b2a70af](https://user-images.githubusercontent.com/14838781/176458692-b394f979-2549-4042-8f53-4a5de39105ec.png)
![4e745d071dfc3cacae6d6735f0b818c](https://user-images.githubusercontent.com/14838781/176458729-689fcdba-b721-4c49-8acf-fc4279b2dd6a.png)

**Comment by jbwang1997** [comment, marked as answer]
It's because we adapt a new coordinate system as detectron2. Can refer to [here](https://mmdetection.readthedocs.io/en/latest/compatibility.html#coordinate-system)
